### PR TITLE
read only option for custom field enhanced

### PR DESCRIPTION
**Pull request by zamandevs**

### CODE REVIEW

**Comment by FaraziF** [comment]
Enhancement request: The required fields will be disabled while the read-only field is enabled.
Screenshot 2021-12-20 at 3 57 18 PM

File: assets/js/frontend-form.js [hunk 1]
```diff
@@ -106,6 +107,14 @@
             });
         },
 
+        handleReadOnly: function(){
+            $('.wpuf-form > .read-only').each( function () {
+                $('input, select, textarea', this).each( function () {
+                     $(this).attr( 'disabled', true );
```
Comment (tareq1988) [inline assets/js/frontend-form.js:113]: I haven't run the code, but we are disabling the fields here. Is there any code to enable the disabled ones again? If it works, no problem.

Comment (zamandevs) [reply assets/js/frontend-form.js:113]: There isn't any code to enable this. as the requirement it is not like a toggle. the field will be disabled if the admin wants to.

**Comment by DenisCGN** [comment]
I like to have that option too, especially to show only the USER_LOGIN without to be able to change it.

**Pull request merged**